docs/content/policy-reference.md

```diff
@@ -919,6 +920,9 @@ conjunction with the `force_cache_duration_seconds` field. If `force_cache` is `
 
 Also, if `force_cache` is `true`, it overrides the `cache` field.
 
+`http.send` only caches responses with the following HTTP status codes: `200`, `203`, `204`, `206`, `300`, `301`,
```

Comment:
Does this have an impact on forced caching as well? I could not discern that from this description, nor the tests. It's probably fine to skip caching of errors even with forced caching, but it would be good if that behavior was mentioned explicitly.

Reply:
So if `force_cache` is `true` and the response status code is not in the list of allowed HTTP status codes, OPA will not cache the response. I'll update the docs to state this more explicitly. In the future if needed we can add a new config option to the built-in to cache irrespective of status code.

Reply:
Updated the doc.

Reply:
Thanks!